econet_base.yaml

```diff
@@ -154,6 +154,32 @@ text_sensor:
     request_once: true
     icon: "mdi:information-box"
     entity_category: "diagnostic"
+  - platform: template
+    name: "Microcontroller Datetime"
+    id: irtc_time
+    icon: mdi:calendar-clock
+    update_interval: 30s
+    entity_category: "diagnostic"
+    disabled_by_default: true
+    lambda: |-
+      const int timeStringSize = 21;
+      if (!((int) id(irtc_seconds).state >= 0)) {  // Return undefined if values haven't been successfully fetched yet
```

Comment:
Is the casting necessary?

Reply:
Probably not? I was aggressively casting this to int since it's always an integer, but I guess this comparison would work without the cast.